cypress/support/functions.ts

```diff
@@ -1,19 +1,25 @@
 cy.functions = {
-    downloadPath: (theFile) => {
-        return Cypress.config('downloadsFolder') + Cypress.env('FILESEP') + theFile;
-    },
+  downloadPath: (theFile) => {
+    return Cypress.config('downloadsFolder') + Cypress.env('FILESEP') + theFile;
+  },
 
   testTabDeselectReportNamed: (nameToSelect) => {
-        cy.get('[data-cy-test="table"]').find('tr').each(function($reportRow) {
-            cy.log('Considering next report');
-            const reportName = $reportRow.find('td').eq(2).text();
-            cy.log('Name of report is: ' + reportName);
-            if(reportName.includes(nameToSelect)) {
-                cy.log('Found checkbox of report with name name, checking it');
-                cy.wrap($reportRow).find('[type=checkbox]').uncheck();
-            };
-            cy.get('[data-cy-test="reportChecked"]:checked').should('have.length', 1);
-            cy.wait(1000);
-        });
-    }
-}
+    cy.get('[data-cy-test="table"]')
+      .find('tr')
+      .each(function ($reportRow) {
+        cy.log('Considering next report');
+        const reportName = $reportRow.find('td').eq(2).text();
+        cy.log('Name of report is: ' + reportName);
+        if (reportName.includes(nameToSelect)) {
+          cy.log('Found checkbox of report with name name, checking it');
+          cy.wrap($reportRow).find('[type=checkbox]').uncheck();
+        }
+        cy.get('[data-cy-test="reportChecked"]:checked').should('have.length', 1);
+        cy.wait(1000);
+      });
+  },
+
+  assertDebugTableLength: (length: number) => {
+    cy.getTableBody().find('tr').should('have.length', length);
+  },
```

Comment:
```suggestion
  assertDebugTableLength: (length: number) => {
    if (length === 0) {
      cy.getTableBody().find('tr').should('not.exist');
    } else {
      cy.getTableBody().find('tr').should('have.length', length);
    }
  },
```

Reply:
Added the change. Conditional branching was suggested by ESLint so I used that instead.